avocado/utils/cpu.py

```diff
@@ -123,15 +123,18 @@ def get_cpu_arch():
                  ('^cpu.*POWER8', 'power8'),
                  ('^cpu.*POWER9', 'power9'),
                  ('^cpu.*PPC970', 'power970'),
-                 ('ARM', 'arm'),
-                 ('^flags.*:.* lm .*', 'x86_64')]
+                 ('(ARM|^CPU implementer|^CPU part|^CPU variant'
+                  '|^Features|^BogoMIPS|^CPU revision)', 'arm'),
```

Comment:
Btw I don't think we should be so lenient towards the values. The less special keys we look for the less possible mistakes we do (any new architecture might reuse the same key)